pkg/cmd/server/api/types.go

```diff
@@ -998,3 +1001,26 @@ type AdmissionConfig struct {
 	// on the master. Order is significant. If empty, a default list of plugins is used.
 	PluginOrderOverride []string
 }
+
+// ClusterResourceOverrideConfig is the configuration for the ClusterResourceOverride
+// admission controller which overrides user-provided container request/limit values.
+type ClusterResourceOverrideConfig struct {
+	unversioned.TypeMeta
+	// Enabled must be true for the plugin to do anything.
+	// The plugin's actions can be disabled per-project with the project annotation
+	// quota.openshift.io/cluster-resource-override-enabled="false", so cluster admins
+	// can exempt infrastructure projects and such from the overrides.
+	Enabled bool
```

Comment:
> So... we keep the Enabled flag?

I see
 1. no entry in the plugin config - disable
 1. no value in location and config - fails validation
 1. no value in location, value in config - this is enabled.
 1. value in location, value in config - fails validation
 1 . value in location, no value in config - this is enabled

Which case is ambiguous?  "value in location, no value in config, zero bytes in file"?  That should fail.